assert/assert_is_error.ts

```diff
@@ -22,14 +22,14 @@ import { stripAnsiCode } from "@std/internal/styles";
  *
  * @typeParam E The type of the error to assert.
  * @param error The error to assert.
- * @param ErrorClass The optional error class to assert.
```

Comment:
I slightly prefer capital `ErrorClass` as it indicates this is a class constructor instead of a class instance.